std/net.zig

```diff
@@ -4,25 +4,22 @@ const assert = std.debug.assert;
 const net = @This();
 const mem = std.mem;
 const os = std.os;
+const system = os.system;
 
-pub const TmpWinAddr = struct {
-    family: u8,
-    data: [14]u8,
-};
-
-pub const OsAddress = switch (builtin.os) {
-    builtin.Os.windows => TmpWinAddr,
-    else => os.sockaddr,
-};
+pub const OsAddress = os.system.sockaddr;
```

Comment:
Should this be a `sockaddr_storage` or perhaps even `sockaddr_any` (which is the normal alias used in C when you also include a `struct sockaddr_un`)

Reply:
`std.os.system.sockaddr` should be called `sockaddr_storage` as it's the union you're describing, although it does not include a `sockaddr_un`.